Add University of Michigan to list of collaborators

eycramer opened the pull request:
## Description

Add UM to list of collaborators
---


## Checklist

- [ ] Specify a proper PR title with your team name.
- [ ] All validation checks ran successfully on your branch. Instructions to run the tests locally is present [here](https://github.com/reichlab/covid19-forecast-hub/wiki/Running-Checks-Locally).

schwarzt175 commented:
Hi Estee,

We can just use "The University of Michigan".  I think the validation checks are already complete.

Tom

eycramer commented:
Sure!

I updated the name and fixed the link. It should be showing up correctly
now in the readme.

Best,
Estee

On Wed, Jul 8, 2020 at 12:32 PM Thomas Schwarz <notifications@github.com>
wrote:

> Hi Estee,
>
> We can just use "The University of Michigan". I think the validation
> checks are already complete.
>
> Tom
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/reichlab/covid19-forecast-hub/pull/772#issuecomment-655656580>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOMG43NRWVCRA6TAOQX7HNTR2SUTZANCNFSM4OUZAXMQ>
> .
>

schwarzt175 commented:
Thank you Estee!

===================
Tom Schwarz
Associate Professor
Department of Physics
University of Michigan
Office: 734-764-8941
Lab: 734-763-7805
schwarzt@umich.edu
===================



On Wed, Jul 8, 2020 at 1:41 PM Estee Cramer <notifications@github.com>
wrote:

> Sure!
>
> I updated the name and fixed the link. It should be showing up correctly
> now in the readme.
>
> Best,
> Estee
>
> On Wed, Jul 8, 2020 at 12:32 PM Thomas Schwarz <notifications@github.com>
> wrote:
>
> > Hi Estee,
> >
> > We can just use "The University of Michigan". I think the validation
> > checks are already complete.
> >
> > Tom
> >
> > —
> > You are receiving this because you modified the open/close state.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/reichlab/covid19-forecast-hub/pull/772#issuecomment-655656580
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AOMG43NRWVCRA6TAOQX7HNTR2SUTZANCNFSM4OUZAXMQ
> >
> > .
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/reichlab/covid19-forecast-hub/pull/772#issuecomment-655660929>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJCRWHXHYVY3EHTGIFKWTZ3R2SVU7ANCNFSM4OUZAXMQ>
> .
>
